app/forms/openid_connect_authorize_form.rb

```diff
@@ -130,6 +131,10 @@ def requested_aal_value
                  :ial2_or_greater?,
                  :ial2_requested?
 
+  def biometric_comparison_required
```

Comment:
if we're making this a method here can we make it a predicate?
```suggestion
  def biometric_comparison_required?
```

Reply:
We can, though I would prefer we keep this and do an alias or something like that. This is overriding the method that is created by the `attr_reader` above.

https://github.com/18F/identity-idp/blob/054b81a29fe40d60a8baae50323fa3d188c39f05/app/forms/openid_connect_authorize_form.rb#L28

I opted not to take this out of the ATTRS array to keep the allowlisted mass assign functionality in place

https://github.com/18F/identity-idp/blob/054b81a29fe40d60a8baae50323fa3d188c39f05/app/forms/openid_connect_authorize_form.rb#L55

I could move it into the not-simple attr though and just have the predicate. WDYT?

Reply:
I'd have it as a not-so-simple attr (maybe clean it up on assignment) and remove the `attr_reader`

Reply:
Addressed in d64fd844a5614d7f1b9efedd7e7e6cc19dc56af0